CRM/Contribute/Form/Contribution/Confirm.php

```diff
@@ -487,8 +487,7 @@ public function buildQuickForm() {
       $this->buildCustom($this->_values['honoree_profile_id'], 'honoreeProfileFields', TRUE, 'honor', $fieldTypes);
     }
     $this->assign('receiptFromEmail', $this->_values['receipt_from_email'] ?? NULL);
-    $amount_block_is_active = $this->get('amount_block_is_active');
```

Comment:
set in preProcess - but preProcess also ensure the _values value is set so why have 2 ways?